src/components/dialog/dialog.tsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import React from 'react';
+import * as RadixDialog from '@radix-ui/react-dialog';
+import { CloseIcon } from '../icons';
+
+import styles from './dialog.module.scss';
+
+export interface DropdownMenuProps extends React.PropsWithChildren {
```

Comment:
please extend radix dialog props and also accept RadixDialog.ContentProps
and bypass all that into Radix to allow better flexibility. see DropDownMenu component for example